### PR TITLE
EE: Add COP1 Instructions to profiler

### PR DESCRIPTION
**Pull request by ssakash**
**Summary of changes:**

* Implement COP1 Instructions for EE profiler
* align a comment , brace before willkuer / turtleli complains.

Thanks to @gregory38 for providing the idea of preventing a possibility of NOP being profiled. Also thanks to rama and refraction for reviewing the change :)

### CODE REVIEW

File: pcsx2/x86/iFPU.cpp [hunk 1]
```diff
@@ -941,6 +957,7 @@ void recCVT_W()
 		DOUBLE::recCVT_W();
 		return;
 	}
+	EE::Profiler.EmitOp(eeOpcode::CVTW);
```
Comment (ssakash) [inline pcsx2/x86/iFPU.cpp]: If we have the following EmitOP() on the top then it'll get calculated twice if ``CHECK_FPU_FULL`` is true as we also have a EmitOP() on recCVT_W on iFPUd.cpp. hence have it below the possible return.

Comment (gregory38) [inline pcsx2/x86/iFPU.cpp]: Put this kind of comment in the code itself ;)

**Comment by gregory38** [comment]
@ssakash when you rebase and change some patches. Could you add some comment on the commit message such as
```
v2: add a comment on function X to explain why A is after B
```
This way it is easier to review.

**Comment by ssakash** [comment]
sure, shouldn't be a problem. I'll do it that way for the any future rebase.

**Pull request merged**